### DISCUSSION TITLE
FastLogin treating as new account when we add one to /premium list

### DISCUSSION THREAD

**Discussion by EliteDenizen** [games647/FastLogin#1243] [General]
Hi! I've setup FastLogin (also using AuthMe Reloaded) on my bungee and 2 bukkit servers (hub and skyblock) and when I joined I had different inventory, money, rank, permissions and stuff and when I did /premium, it just treated me as new premium player (the autologin and autoregister parts work fine but how do I make it treat same accounts, I've premium Uuid on true) What steps can I do? Also, I get this in my console

[LuckPerms] LuckPerms already has data for player 'Araxion' - but this data is stored under a different UUID.
[00:35:58 WARN]: [LuckPerms] 'Araxion' has previously used the unique ids [MY UUID] but is now connecting with 'Another My UUID'
[00:35:58 WARN]: [LuckPerms] The UUID the player is connecting with now is NOT Mojang-assigned (type 3). This implies that THIS server is not authenticating correctly, but one (or more) of the other servers/proxies in the network are.
[00:35:58 WARN]: [LuckPerms] If you're using BungeeCord/Velocity, please ensure that IP-Forwarding is setup correctly on all of your backend servers!
[00:35:58 WARN]: [LuckPerms] See here for more info: https://luckperms.net/wiki/Network-Installation#pre-setup

When joined after unpremium myself and this when I first joined after premium:
[FastLogin] Received proxy id: (UUID here) that doesn't exist in the proxy file

**Comment by ElTuna** [comment]
I´m having the same issue...
My players have an inventory, homes, etc when they join the first time...
And after running `/premium` and they leave and join the server again they appear as **new** users... They have a NEW inventory and everything... how to fix that?? (I have `premiumUuid: true`)

**Reply by TuxCoding** [reply]
If the you want to the player access to the same data, you need to disable premiumUUID. Then, then UUID doesn't get switched from your existing data.

**Reply by ElTuna** [reply]
and that way they will also keep their inventory and things when change their premium name?

**Reply by ElTuna** [reply]
another question... if i change the premium UUID setting now in production... will my premium players have reseted their actual premium data??

**Reply by TuxCoding** [reply]
>and that way they will also keep their inventory and things when change their premium name?

No that requires to enable premium UUID. Otherwise the data needs to be converted and that's only possible in limited ways like only vanilla data and therefore currently not implemented.

>another question... if i change the premium UUID setting now in production... will my premium players have reseted their actual premium data??

The plugin doesn't convert any existing data, so it only set the UUID of the player instance currently playing.
